tests/test_find_django.py

```diff
@@ -26,7 +27,7 @@
     'code_annotations.find_django.DjangoSearch',
     get_models_requiring_annotations=DEFAULT
 )
-def test_find_django_simple_success(**kwargs):
+def test_find_django_simple_success(**kwargs: t.Any) -> None:
```

Comment:
Most, if not all, of these `t.Any` params can be `MagicMock` as they are in the other file.